docs/decisions/0001-course-experience.rst

```diff
@@ -0,0 +1,283 @@
+Courses units in LMS
+--------------
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+NELP business case requires multiples features that are not part
+of open-edx platform core, and it's necessary a new implementation
+that allows users to report and share their opinions about the
+course content and experience during the course.
+
+Decisions
+=========
+
+1. Create a course experience module.
+2. Create ``RateCourse`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
```

Comment:
Foreign key to the user model or only to the username field of the model user?

Reply:
Foreign key to user model